src/viztracer/viztracer.py

```diff
@@ -43,7 +43,8 @@ def __init__(self,
                  dump_raw: bool = False,
                  sanitize_function_name: bool = False,
                  output_file: str = "result.json",
-                 plugins: Sequence[Union[VizPluginBase, str]] = []) -> None:
+                 plugins: Sequence[Union[VizPluginBase, str]] = [],
+                 process_name: Optional[str] = None) -> None:
```

Comment:
Put `process_name` before `output_file` -> there's no strict order here, but I always feel like `output_file` and `plugins` should be at the end of the list.